core/src/main/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderConfiguration.kt

```diff
@@ -40,6 +41,12 @@ class PublicationConfiguration(
     val maxBatchSize: Int = 100,
 )
 
+class EventPublicationConfiguration(
+    val maxBatchSizeInBytes: Long = 1_024 * 1_024,
```

Comment:
Why do we need to ask the user about that? We have a cradle in our common factory. It contains all the required information about the event batch size